packages/react-components/react-message-bar-preview/src/components/MessageBarActions/MessageBarActions.test.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { isConformant } from '../../testing/isConformant';
+import { MessageBarActions } from './MessageBarActions';
+import { MessageBarActionsProps } from './MessageBarActions.types';
+
+describe('MessageBarActions', () => {
+  isConformant<MessageBarActionsProps>({
+    Component: MessageBarActions,
+    displayName: 'MessageBarActions',
+    disabledTests: [
+      // TODO: having problems due to the fact root of DialogTitle is Fragment
```

Comment:
What?

Reply:
I copied that config from DialogTitle, will remove in another PR - the test itself should still be disabled tho